domain-management/src/main/java/org/jboss/as/domain/management/security/adduser/AddUser.java

```diff
@@ -405,6 +413,12 @@ public String instructions() {
             public String instructions() {
                 return DomainManagementLogger.ROOT_LOGGER.argHelp();
             }
+        },
+        DISPLAY_SECRET("-sv", "--secret") {
```

Comment:
It comes from a "**s**ecret **v**alue", as also _--help_ output would suggest you.
See [DomainManagementLogger.java#L1447](https://github.com/hradecek/wildfly-core/blob/ff3cda09e37c52bd79b635b795e85a35fbcc7095/domain-management/src/main/java/org/jboss/as/domain/management/logging/DomainManagementLogger.java#L1447) for help string.